StalinSort implemented in python

SleepingNerd opened the pull request:
Sorting algorithm where unsorted elements are eliminated.

SleepingNerd committed:
StalinSort implemented in python

Sorting algorithm where unsorted elements are eliminated.

pep8speaks commented:
Hello @SleepingNerd! Thanks for updating this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:



There are currently no PEP 8 issues detected in this Pull Request. Cheers! :beers: 

##### Comment last updated at 2022-02-26 18:17:23 UTC

SleepingNerd committed:
Cleaned up version of StalinSort.py

Fixed:
Line 22:5: E303 too many blank lines (2)
Line 25:1: E305 expected 2 blank lines after class or function definition, found 1

SleepingNerd committed:
Final version of StalinSort.py

* Added a docstring
